src/main/scala/images/Program.scala

```diff
@@ -3,20 +3,20 @@ def image_program() = {
   val imageFilePath = "src/resources/image.png"
   val outputFilePath = "output.png"
 
-  flipHorizontal(imageFilePath, outputFilePath)
-  grayscale(outputFilePath, outputFilePath)
-  rotateRight(outputFilePath, outputFilePath)
+  //flipHorizontal(imageFilePath, outputFilePath)
+  //grayscale(outputFilePath, outputFilePath)
+  //rotateRight(outputFilePath, outputFilePath)
 
   // Instead of the code above, we want to use function composition to write the following:
-  // save(
-  //   rotateRight(
-  //     grayscale(
-  //       flipHorizontal(
-  //         load(imageFilePath)
-  //       )
-  //     )
-  //   ),
-  //   outputFilePath
-  // )
+  save(
+    rotateRight(
+      grayscale(
+        flipHorizontal(
+          load(imageFilePath)
+        )
+      )   
+    ),
+    outputFilePath
+  )
```

Comment:
I do think there wasn't a lot of ways you could've done this; my approach is almost exactly the same.